preexisting-btrfs.sh

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2023  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Petr Beranek <pberanek@redhat.com>
+
+# This test covers https://bugzilla.redhat.com/show_bug.cgi?id=2139169 and
+# https://bugzilla.redhat.com/show_bug.cgi?id=2139166
+#
+# Disk image was created via the following commands:
+#   dd if=/dev/zero of=disk-a.img bs=1M seek=10240 count=1
+#   sudo losetup -f disk-a.img
+#   sudo parted --script /dev/<device> mklabel gpt  # use 'sudo losetup -l' to
+#                                                   # discover actual <device> name
+#   sudo parted --script /dev/<device> mkpart p1 btrfs 0% 100%
+#   sudo mkfs.btrfs /dev/<device><partition>  # use 'lsblk' to discover actual
+#                                             # <partition> name
+#   sudo losetup -d /dev/<device>
+#   tar caSf preexisting-btrfs.tar.xz disk-a.img
+
+TESTTYPE="btrfs storage"
```

Comment:
Please add a comment explaining why we want to run the test on rhel too. The first thought about this place would be usually: btrfs+rhel=nope so why is `skip-on-rhel` missing here? Something like "enabled on rhel since it must be able to install over btrfs", perhaps...